src/test/scala/org/neo4j/spark/Neo4jGraphScalaTSE.scala

```diff
@@ -0,0 +1,86 @@
+package org.neo4j.spark
+
+import org.apache.spark.graphx.{Edge, Graph, VertexId}
+import org.apache.spark.rdd.RDD
+import org.junit.Assert._
+import org.junit._
+
+import scala.collection.JavaConverters._
+
+object Neo4jGraphScalaTSE {
+
+}
+
+/**
+  * @author mh
+  * @since 17.07.16
+  */
+class Neo4jGraphScalaTSE extends SparkConnectorScalaBaseTSE {
+  val FIXTURE: String = "CREATE (s:A {a:0})-[r:REL {foo:'bar'}]->(t:B {b:1}) RETURN id(s) AS source, id(t) AS target"
+
+  private var source: Long = _
+  private var target: Long = _
+
+  @Before
+  @throws[Exception]
+  def setUp {
```

Comment:
should we do these tests with larger numbers that exceed partition sizes?

Reply:
ping @conker84

Reply:
Yes I'll do that in a separate issue ok? I created #123